serpico.rb

```diff
@@ -5,6 +5,12 @@
 require "./server.rb"
 config_options = JSON.parse(File.read('./config.json'))
 
+# SSL Ciphers
+CIPHERS = ['ECDHE-RSA-AES128-GCM-SHA256','ECDHE-RSA-AES256-GCM-SHA384',
```

Comment:
Any chance we can make it customizable variable. that ways we leave the headache of matching the ciphers as per company requirements. SSL requirements keep changing and we don't want to be the one holding back anyone.